### PR TITLE
Client Improvements

### PR DESCRIPTION
**Pull request by NeonKirill**
- Minifications
- Naming fixes
- Bug fixes

### CODE REVIEW

File: .github/workflows/publish_release.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        with:
+          python-version: 3.8
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+
+      - name: Build Distribution Packages
+        run: |
+          python setup.py bdist_wheel
+      - name: Publish to Test PyPI
```
Comment (NeonDaniel) [inline .github/workflows/publish_release.yml:37]: I don't see a `setup.py` file; is this pip installable? If not, the version tagging is worth keeping, but the `build_and_publish` action could be removed

File: .github/workflows/publish_test_build.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+name: Publish Alpha Build
+on:
+  push:
+    branches:
+      - dev
+    paths-ignore:
+      - 'version.py'
+
+jobs:
+  build_and_publish:
```
Comment (NeonDaniel) [inline .github/workflows/publish_test_build.yml:12]: Same as above; this automation can be removed if PyKlatchat can't be built

File: scripts/files_manipulator.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+from abc import ABC, abstractmethod
+from os import listdir
+from os.path import isfile, join
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(level=logging.INFO)
+
+
+class FilesManipulator(ABC):
```
Comment (NeonDaniel) [inline scripts/files_manipulator.py:41]: We should consider moving this under neon_utils in the future

File: scripts/files_manipulator.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        """ Implement to handle valid files """
+        pass
+
+    def on_failed_file(self, file_path):
+        """ Implement to handle failed files """
+        pass
+
+    def walk_tree(self, folder: str = ''):
+        """ Walks towards specified folder and processes files"""
+        if folder:
```
Comment (NeonDaniel) [inline scripts/files_manipulator.py:74]: `os.walk` might provide some of this logic for you to simplify this method

**Review by NeonDaniel — CHANGES_REQUESTED** [review]
Few comments in automations, otherwise looks good to me

File: scripts/files_manipulator.py (hunk 4 above)
Comment (NeonKirill) [reply scripts/files_manipulator.py:74]: > os.walk might provide some of this logic for you to simplify this method
os.walk takes into account each of the layers without subfolders, so I had to made the one that does the job

File: scripts/files_manipulator.py (hunk 3 above)
Comment (NeonKirill) [reply scripts/files_manipulator.py:41]: agree

File: .github/workflows/publish_release.yml (hunk 1 above)
Comment (NeonKirill) [reply .github/workflows/publish_release.yml:37]: > I don't see a `setup.py` file; is this pip installable? If not, the version tagging is worth keeping, but the `build_and_publish` action could be removed

I has setup.py in root

Comment (NeonDaniel) [reply .github/workflows/publish_release.yml:37]: Can you add build tests to unit tests (or another workflow)? If they aren't passing, PyPI publication can be omitted here and left for another PR. This workflow would close #20
```yaml
  build_tests:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Setup Python
        uses: actions/setup-python@v1
        with:
          python-version: 3.8
      - name: Install Build Tools
        run: |
          python -m pip install build wheel
      - name: Build Distribution Packages
        run: |
          python setup.py bdist_wheel
```

**Pull request merged**